locales/sk/json.json

```diff
@@ -56,58 +56,58 @@
+    "Are you sure you want to detach this resource?": "Ste si istí, že chcete túto položku odpojiť?",
+    "Are you sure you want to force delete the selected resources?": "Ste si istí, že chcete permanentne odstrániť vybrané položky?",
+    "Are you sure you want to force delete this resource?": "Ste si istí, že chcete permanentne odstrániť túto položku?",
     "Are you sure you want to log out?": "Naozaj sa chcete odhlásiť?",
     "Are you sure you want to remove this item?": "Naozaj chcete odstrániť túto položku?",
-    "Are you sure you want to restore the selected resources?": "Ste si istý, že chcete obnoviť vybrané zdroje?",
-    "Are you sure you want to restore this resource?": "Ste si istí, že chcete obnoviť tento zdroj?",
+    "Are you sure you want to restore the selected resources?": "Ste si istý, že chcete obnoviť vybrané položky?",
+    "Are you sure you want to restore this resource?": "Ste si istí, že chcete obnoviť túto položku?",
     "Are you sure you want to resume your subscription?": "Naozaj chcete obnoviť svoje predplatné?",
     "Are you sure you want to run this action?": "Ste si istí, že chcete spustiť túto akciu?",
     "Are you sure you want to set this payment method as your default?": "Naozaj chcete nastaviť tento spôsob platby ako predvolený?",
-    "Are you sure you want to stop impersonating?": "Naozaj chcete prestať s odcudzením identity?",
-    "Are you sure you would like to delete this API token?": "Ste si istí, že chcete tento token API odstrániť?",
-    "Are you sure you would like to leave this team?": "Ste si istý, že by ste chceli opustiť tento tím?",
-    "Are you sure you would like to remove this person from the team?": "Ste si istý, že by ste chceli odstrániť túto osobu z tímu?",
+    "Are you sure you want to stop impersonating?": "Chcete ukončiť prihlásenie pod inou identitou?",
+    "Are you sure you would like to delete this API token?": "Ste si istí, že chcete tento API token odstrániť?",
+    "Are you sure you would like to leave this team?": "Ste si istý, že chcete opustiť tento tím?",
+    "Are you sure you would like to remove this person from the team?": "Ste si istý, že chcete odstrániť túto osobu z tímu?",
     "Are you sure you would like to switch billing plans?": "Naozaj chcete zmeniť tarify?",
     "Argentina": "Argentína",
     "Armenia": "Arménsko",
     "Aruba": "Aruba",
-    "Attach": "Pripojiť",
-    "Attach & Attach Another": "Pripojiť A Pripojiť Ďalšie",
-    "Attach :resource": "Priložiť :resource",
-    "Attach files by dragging & dropping, selecting or pasting them.": "Pripojte súbory pretiahnutím, výberom alebo ochutnaním.",
+    "Attach": "Pridať",
+    "Attach & Attach Another": "Pridať a pokračovať v pridávaní",
+    "Attach :resource": "Pridať :resource",
+    "Attach files by dragging & dropping, selecting or pasting them.": "Pripojte súbory presunutím myšou, výberom alebo vložením.",
     "Attendance": "Dochádzka",
-    "August": "augusta",
+    "August": "August",
```

Comment:
The month name also needs to be added to the `_excludes.json` file, otherwise the status will show as requiring translation.